sys/mips/include/pte.h

```diff
@@ -194,10 +194,10 @@ typedef uint64_t pd_entry_t;
  *
  * Upper bits of a 64 bit PTE:
  *
- *   63-62   61-60  59   58 -- 56    55   54   53
- *   ---------------------------------------------
- *  |  RG  |      | SV | PG SZ IDX | MN | W  | RO |
```

Comment:
Is this bit no longer used? If so we should unallocate it upstream.

Reply:
I think it was used only in the "new" pmap, which got removed.